src/GNSS.cpp

```diff
@@ -41,13 +50,14 @@ void GNSS_srv(){
 }
 
 float GNSS_calcDistance(float targetLat, float targetLon){
-    if(myFix == 0)
+   if(myFix == 0)
         return -1.0f;
-
-    float rlat1 = 3.14f * myLat / 180.0f;
-    float rlat2 = 3.14f * targetLat / 180.0f;
+    
+    /*
+    float rlat1 = 3.141592f * myLat / 180.0f;
```

Comment:
You have Pi as macro definition - use it

Reply:
Removed this part of code